pkg/server/plugin/upstreamauthority/gcpcas/gcpcas_test.go

```diff
@@ -185,7 +185,7 @@ func TestGcpCAS(t *testing.T) {
 }
 
 func generateCert(t *testing.T, cn string, issuer *x509.Certificate, issuerKey crypto.PrivateKey, ttlInHours int, keyfn func(testing.TB) *ecdsa.PrivateKey) (*x509.Certificate, crypto.PrivateKey, error) {
-	priv := keyfn(t)
+	keyPair := keyfn(t)
```

Comment:
maybe `privateKey`?

Reply:
I thought that looked confusing for line 208, it'd be
```go
privateKey.Public()
```
instead of
```go
keyPair.Public()
```